userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -319,11 +319,17 @@ bool docker_async_source::parse_docker(std::string &container_id, sinsp_containe
 			Json::Value img_root;
 			if(reader.parse(img_json, img_root))
 			{
+			    string prevdigest = "";
+			    bool foundsingledigest = true;
 				for(const auto& rdig : img_root["RepoDigests"])
 				{
 					if(rdig.isString())
 					{
 						string repodigest = rdig.asString();
+						string currentdigest = repodigest.substr(repodigest.find("@")+1);
```

Comment:
```suggestion
						string currentdigest = repodigest.substr(repodigest.find('@')+1);
```

(same for the other calls; yeah, I know the precedent of using "@" is there but let's not follow it but maybe fix it in existing code instead)

Reply:
@gnosek Sorry I did not understand. Are you saying we should not use `@` to retrieve the digest?

Reply:
I mean using `'@'` (a single character) instead of `"@"` (a one-character string)

Reply:
ah gotcha 👍